source/base/MessageHandler.cpp

```diff
@@ -109,15 +110,27 @@ void MessageHandler::handleMessageReceived(Swift::Message::ref message)
             isGroupMessage = true;
         }
 
+        QString messageId = QString::fromStdString(message->getID());
+		if (messageId.length() == 0)
+        {
+            // No message id, try xep 0359
+            std::shared_ptr<StanzaIdPayload> stanzaId = message->getPayload<StanzaIdPayload>();
+            if (stanzaId != nullptr)
+            {
+                QString messageId = QString::fromStdString(stanzaId->getId());
+
```

Comment:
This messageId gets out of scope after the closing curly bracket.
Remove the empty line.

Reply:
Yes, I just realized that a few minutes ago...